crates/bevy_math/src/bounding/bounded2d/mod.rs

```diff
@@ -277,6 +287,19 @@ mod aabb2d_tests {
         assert!(!shrunk.contains(&a));
     }
 
+    #[test]
+    fn scale_around_center() {
+        let a = Aabb2d {
+            min: Vec2::new(-1., -1.),
+            max: Vec2::new(1., 1.),
```

Comment:
Similarly to the previous comment, there are constants for this:
```suggestion
            min: Vec2::NEG_ONE,
            max: Vec2::ONE,
```